diff -uprN helm-3.11.1/cmd/helm/root_unix.go helm-3.11.1-patch/cmd/helm/root_unix.go
--- helm-3.11.1/cmd/helm/root_unix.go	2023-02-09 01:02:33.000000000 +0800
+++ helm-3.11.1-patch/cmd/helm/root_unix.go	2023-02-20 17:38:55.495829062 +0800
@@ -40,7 +40,13 @@ func checkPerms() {
 			// can proceed happily without a KUBECONFIG, so this is not a fatal error.
 			return
 		}
-		kc = filepath.Join(u.HomeDir, ".kube", "config")
+		kf := "/etc/kubernetes/admin.conf"
+		_, err2 := os.Stat(kf)
+		if err2 != nil {
+			kc = filepath.Join(u.HomeDir, ".kube", "config")
+		} else {
+			kc = kf
+		}
 	}
 	fi, err := os.Stat(kc)
 	if err != nil {
