diff -uprN containerd-1.6.16/cmd/containerd-stress/main.go containerd-1.6.16-patch/cmd/containerd-stress/main.go
--- containerd-1.6.16/cmd/containerd-stress/main.go	2023-01-29 02:10:17.000000000 +0800
+++ containerd-1.6.16-patch/cmd/containerd-stress/main.go	2023-02-09 16:47:50.944887995 +0800
@@ -136,7 +136,7 @@ func main() {
 		},
 		cli.StringFlag{
 			Name:  "image,i",
-			Value: "docker.io/library/alpine:latest",
+			Value: "boot.powerk8s.cn/library/alpine:latest",
 			Usage: "image to be utilized for testing",
 		},
 		cli.BoolFlag{
diff -uprN containerd-1.6.16/integration/client/image_test.go containerd-1.6.16-patch/integration/client/image_test.go
--- containerd-1.6.16/integration/client/image_test.go	2023-01-29 02:10:17.000000000 +0800
+++ containerd-1.6.16-patch/integration/client/image_test.go	2023-02-09 16:47:04.884217674 +0800
@@ -79,7 +79,7 @@ func TestImageIsUnpacked(t *testing.T) {

 func TestImagePullWithDistSourceLabel(t *testing.T) {
 	var (
-		source   = "registry.k8s.io"
+		source   = "boot.powerk8s.cn"
 		repoName = "pause"
 		tag      = "3.6"
 	)
@@ -231,7 +231,7 @@ func TestImageUsage(t *testing.T) {
 func TestImageSupportedBySnapshotter_Error(t *testing.T) {
 	var unsupportedImage string
 	if runtime.GOOS == "windows" {
-		unsupportedImage = "registry.k8s.io/pause-amd64:3.2"
+		unsupportedImage = "boot.powerk8s.cn/pause-amd64:3.2"
 	} else {
 		unsupportedImage = "mcr.microsoft.com/windows/nanoserver:1809"
 	}
diff -uprN containerd-1.6.16/integration/common.go containerd-1.6.16-patch/integration/common.go
--- containerd-1.6.16/integration/common.go	2023-01-29 02:10:17.000000000 +0800
+++ containerd-1.6.16-patch/integration/common.go	2023-02-09 16:47:29.534576173 +0800
@@ -47,12 +47,12 @@ var (

 func initImages(imageListFile string) {
 	imageList = ImageList{
-		Alpine:           "ghcr.io/containerd/alpine:3.14.0",
-		BusyBox:          "ghcr.io/containerd/busybox:1.28",
-		Pause:            "registry.k8s.io/pause:3.6",
-		ResourceConsumer: "registry.k8s.io/e2e-test-images/resource-consumer:1.10",
-		VolumeCopyUp:     "ghcr.io/containerd/volume-copy-up:2.1",
-		VolumeOwnership:  "ghcr.io/containerd/volume-ownership:2.1",
+		Alpine:           "boot.powerk8s.cn/containerd/alpine:3.14.0",
+		BusyBox:          "boot.powerk8s.cn/containerd/busybox:1.28",
+		Pause:            "boot.powerk8s.cn/pause:3.8",
+		ResourceConsumer: "boot.powerk8s.cn/e2e-test-images/resource-consumer:1.10",
+		VolumeCopyUp:     "boot.powerk8s.cn/containerd/volume-copy-up:2.1",
+		VolumeOwnership:  "boot.powerk8s.cn/containerd/volume-ownership:2.1",
 	}

 	if imageListFile != "" {
diff -uprN containerd-1.6.16/pkg/cri/config/config_unix.go containerd-1.6.16-patch/pkg/cri/config/config_unix.go
--- containerd-1.6.16/pkg/cri/config/config_unix.go	2023-01-29 02:10:17.000000000 +0800
+++ containerd-1.6.16-patch/pkg/cri/config/config_unix.go	2023-02-09 16:47:04.864217385 +0800
@@ -92,7 +92,7 @@ func DefaultConfig() PluginConfig {
 			TLSKeyFile:  "",
 			TLSCertFile: "",
 		},
-		SandboxImage:                     "registry.k8s.io/pause:3.6",
+		SandboxImage:                     "boot.powerk8s.cn/pause:3.8",
 		StatsCollectPeriod:               10,
 		SystemdCgroup:                    false,
 		MaxContainerLogLineSize:          16 * 1024,
diff -uprN containerd-1.6.16/pkg/cri/config/config_windows.go containerd-1.6.16-patch/pkg/cri/config/config_windows.go
--- containerd-1.6.16/pkg/cri/config/config_windows.go	2023-01-29 02:10:17.000000000 +0800
+++ containerd-1.6.16-patch/pkg/cri/config/config_windows.go	2023-02-09 16:47:04.864217385 +0800
@@ -52,7 +52,7 @@ func DefaultConfig() PluginConfig {
 			TLSKeyFile:  "",
 			TLSCertFile: "",
 		},
-		SandboxImage:              "registry.k8s.io/pause:3.6",
+		SandboxImage:              "boot.powerk8s.cn/pause:3.8",
 		StatsCollectPeriod:        10,
 		MaxContainerLogLineSize:   16 * 1024,
 		MaxConcurrentDownloads:    3,
